sample-apps/auto-instrumentation-java-spark/src/main/java/com/amazon/sampleapp/Config.java

```diff
@@ -0,0 +1,69 @@
+package com.amazon.sampleapp;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.io.FileInputStream;
+import java.util.Map;
+import java.util.ArrayList;
+import org.yaml.snakeyaml.Yaml;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Config {
+    private static final Logger logger = LogManager.getLogger();
+    private static String configHost;
+    private static String configPort;
+    private static int configInterval;
+    private static int configTime;
+    private static int configHeap;
+    private static int configThreads;
+    private static int configCpu;
+    private static ArrayList<String> configSamplePorts;
+    public Config() {
+        //reading in values from config yaml file
+        Yaml yaml = new Yaml();
+        InputStream inputStream = getClass().getResourceAsStream("/config.yaml");
+        Map<String, Object> variables = yaml.load(inputStream);
+        configHost = (String) variables.get("Host");
+        configPort = (String) variables.get("port");
```

Comment:
```suggestion
        configPort = (String) variables.get("Port");
```
With the previous suggestion, this should also be updated